packages/client/screens/SigningInScreen.tsx

```diff
@@ -1,19 +1,39 @@
-import { SafeAreaView, Text, useSx, View } from 'dripsy';
-import React from 'react';
+import { useSelector } from '@xstate/react';
+import { Button, SafeAreaView, Text, useSx, View } from 'dripsy';
+import * as Google from 'expo-auth-session/providers/google';
+import * as WebBrowser from 'expo-web-browser';
+import React, { useState } from 'react';
 import { Controller, useForm } from 'react-hook-form';
 import { TouchableOpacity } from 'react-native';
+import invariant from 'tiny-invariant';
 import * as z from 'zod';
-import { useSelector } from '@xstate/react';
 import { AppScreen, TextField } from '../components/kit';
 import { useAppContext } from '../contexts/AppContext';
+import { sendAuthenticateWithGoogleAccount } from '../services/AuthenticationService';
 import { SigningInScreenProps } from '../types';
 
+WebBrowser.maybeCompleteAuthSession();
+
 interface SigningInFormFieldValues {
     email: string;
     password: string;
 }
 
 const SigningInScreen: React.FC<SigningInScreenProps> = ({ navigation }) => {
+    const [accessToken, setAccessToken] = useState<string | undefined>(
+        undefined,
+    );
+    const [request, response, promptAsync] = Google.useAuthRequest({
+        expoClientId:
+            '326703248925-qg0i9ig0b4gqvkpmpfd665d5un082adj.apps.googleusercontent.com',
+        iosClientId:
+            '326703248925-n6knfko9tffasgpq3ffbv8i8qj7vpcgg.apps.googleusercontent.com',
+        androidClientId:
+            '326703248925-fq6r2lio64q1fg0ap6hj6uvm09raf9cd.apps.googleusercontent.com',
+        webClientId:
+            '326703248925-sn4nvaq3vkjoc1chuju6binov8ha3jl3.apps.googleusercontent.com',
+    });
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I'm happy to see how Expo solves social authentication! I can't wait to see how this hook can be used with XState.